recipes/validators/meta.yaml

```diff
@@ -36,22 +36,23 @@ requirements:
   build:
```

Comment:
You can remove the `requirements/build` section altogether when we only have `requirements/host`.